contents/uix/accessibility.md

```diff
@@ -0,0 +1,174 @@
+
+> Taken from [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)
+
+## Why is accessibility important?
+
+##### For the developer
+1. **Accessibility improves everyone's UX**. [W3C](https://www.w3.org/standards/webdesign/accessibility) notes that accessibility practices are good design practices in general. So, improving accessibility brings benefits to everyone.
+2. **Accessibility is a right**. Caring about accessibility demonstrates good ethics/morals.
+3. **Accessibility improves SEO**. Semantic HTML (which improves accessibility) also improves SEO, making your website more discoverable and marketable to new users.
+4. **Accessibility helps you increase your user base**. According to the [World Health Organization world report on disability](https://www.who.int/disabilities/world_report/2011/report/en/), about 15% of the world's population live with some form of disability. That makes up more than 1 billion people. It is a significant population of users, so improving accessibility can help you reach more people.
+5. **Accessibility is part of the law in some places**. In [some countries](https://www.w3.org/WAI/policies/), abiding by accessibility guidelines is mandatory. Be careful not to break the law!
+
+##### For the user
+Accessibility is important for people with disabilities to access the Web. There are many kinds of disabilities, including:
+   * Visual (partial blindness, full blindness, color blindness, cataract, glaucoma, etc.)
+   * Auditory (hard of hearing, deafness, etc.)
+   * Cognitive (ADHD, autism spectrum, etc.)
+   * Mobility (quadriplegia, muscular dystrophy, etc.)
+
+At the same time, accessibility also benefits people _without_ disabilities, for example:
+   * People using mobile phones, smart watches, smart TVs, and other devices with small screens, different input modes, etc.
+   * Older people with changing abilities due to ageing
+   * People with "temporary disabilities" such as a broken arm or lost glasses
+   * People with "situational limitations" such as in bright sunlight or in an environment where they cannot listen to audio
+   * People using a slow Internet connection, or who have limited or expensive bandwidth
+
+## How can I improve the accessibility of my website?
+
+There are many ways to improve the accessibility of your website. Below are some tips, based on the type of disabilities they address.
+
+### Visual
+Users with visual impairments rely on assistive technologies such as a magnifier or a screen reader.
+
+  * Use more visual indicators to convey a message.
+  
+  Let us look at the Facebook sign up page. Suppose I want to sign up for a new account, but I have not put in all the necessary information. So, it is supposed to tell me that my attempt to sign up has failed.
```

Comment:
Indentation doesn't seem right.
![image](https://user-images.githubusercontent.com/1673303/54484558-fba0cf80-48a3-11e9-85ce-428920e85348.png)

Perhaps not use bullet points for this case? Example:
>Users with visual impairments rely on assistive technologies such as a magnifier or a screen reader.
>### Bast practice: use more visual indicators to convey a message
>Let us look at the Facebook sign up page. Suppose I want to sign up for a new account, but I have not put in all the necessary information. So, it is supposed to tell me that my attempt to sign up has failed.